src/components/chips/js/chipsController.js

```diff
@@ -72,6 +72,13 @@ function MdChipsCtrl ($scope, $mdConstant, $log, $element, $timeout) {
    * @type {boolean}
    */
   this.useMdOnAppend = false;
+
+  /**
+   * Whether to use the mdOnSelect expression to notify the component's user
+   * after selecting a chip from the list.
+   * @type {boolean}
+   */
+  this.useMdOnSelect = false;
```

Comment:
This is not a good convention
*   `useMdOnAppend` => `useOnAppend`
*   `useMdOnRemove` => `useOnRemove`
*   `useMdOnSelectExpression` => `useOnSelectExpression`